[CC-14584] AC Doc Documentation Add categories to discount query parameters (#649)
spryker/spryker-docs: docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md

On the **Voucher code** tab, do the following:
1. Enter the **QUANTITY** for voucher codes you want to generate.

> 2. *Optional*: In **CUSTOM CODE**, enter any text that you would like to precede the randomly generated characters in each voucher code. You can also use the placeholder ***[code]*** to specify the location.

Reviewer: ```suggestion
2. *Optional*: In **CUSTOM CODE**, enter any text that you would like to precede the randomly generated characters in each voucher code. You can also use the placeholder *[code]* to specify the location.
```

{% endinfo_block %}

**Promotional product discount**

> This discount type lets you discount a specific product at a fixed quantity whenever the discount conditions are met. A common use case is for a "buy x, get y" discount where a customer gets an item for free when they buy a certain product or spend a certain amount. The product you are promoting goes in the **Abstract product SKU** field. The **Quantity** field represents the limit on the number of items eligible for the discount. When a customer fulfills the discount conditions, the promotional product is automatically merchandised below the items in a customer's cart. The customer can either add the product to the cart from this widget or any other place in the shop, and the discount will apply. It is important to note that the promotional product discount will only apply if the product is added to the cart after meeting the discount's conditions. If the product is already in the cart before the discount conditions are met, the customer will need to remove and re-add it.

Reviewer: ```suggestion
This discount type lets you discount a specific product at a fixed quantity whenever the discount conditions are met. A common use case is for a "buy x, get y" discount where a customer gets an item for free when they buy a certain product or spend a certain amount. The product you are promoting goes in the **Abstract product SKU** field. The **Quantity** field represents the limit on the number of items eligible for the discount. When a customer fulfills the discount conditions, the promotional product is automatically merchandised below the items in a customer's cart. The customer can either add the product to the cart from this widget or any other place in the shop, and the discount will apply. It is important to note that the promotional product discount only applies if the product is added to the cart after meeting the discount's conditions. If the product is already in the cart before the discount conditions are met, the customer needs to remove and re-add it.
```

| VALUE | DESCRIPTION |
| --- | --- |
| Fields | The available fields may include SKU, item-price, item-quantity, or a variety of attributes (for example, **currency** on the image above). |
| Operator | Operator compares the value of a field on the left with the value(s) on the right (for example, equals (=), greater than (>)). If the expression evaluates to true, the discount can be applied (operator is **equal** on the preceding image). |
| Value | Value types must match the selected field. The asterisk (*) matches all possible values (on the preceding image, the value is **Swiss Franc**).|
| Combine Conditions | 'AND' and 'OR' operators are used to combine conditions (**AND** on the preceding image). |

> |Grouping | When building more complex queries, conditions may be grouped inside parentheses '( )'.|

Reviewer 2: ```suggestion
|Grouping | When building more complex queries, conditions may be grouped inside parentheses '( )'. Because discount calculations and conditions are applied per item, it is not possible to use groups with AND that each contain SKU parameters. |
```

> |Grouping | When building more complex queries, conditions may be grouped inside parentheses '( )'. Because discount calculations and conditions are applied per item, it is not possible to use groups with 'AND' that each contain SKU parameters. |

Reviewer: I don't quite get this part "that each contain SKU parameters." Could you please rephrase?

Author: Rewrote it to: "it is not possible to use groups with ‘AND’ where each group contains at least one SKU-based rule."
